src/discord-cluster-manager/leaderboard_db.py

```diff
@@ -467,7 +468,7 @@ def get_leaderboard_submissions(
                 JOIN leaderboard.leaderboard l ON s.leaderboard_id = l.id
                 JOIN leaderboard.user_info ui ON s.user_id = ui.id
                 WHERE l.name = %s
-                    AND r.runner = %s
+                    AND LOWER(r.runner) = LOWER(%s)
```

Comment:
this seems like a ticking time bomb. Can we insure that runner are consistent in the db instead?